network/datadog_checks/network/network.py

```diff
@@ -595,6 +589,46 @@ def _get_net_proc_base_location(proc_location):
             net_proc_base_location = proc_location
         return net_proc_base_location
 
+    def _read_int_file(self, file_location):
+        try:
+            with open(file_location, 'r') as f:
+                try:
+                    value = int(f.read().rstrip())
+                    return value
+                except ValueError:
+                    self.log.debug("Content of %s is not an integer", file_location)
+        except IOError as e:
+            self.log.debug("Unable to read %s, skipping %s.", file_location, e)
+            return None
+
+    def _get_iface_sys_metrics(self, custom_tags):
+        sys_net_location = '/sys/class/net'
+        sys_net_metrics = ['mtu', 'tx_queue_len']
+        try:
+            ifaces = os.listdir(sys_net_location)
+        except OSError as e:
+            self.log.debug("Unable to list %s, skipping %s.", sys_net_location, e)
+            return None
+        for iface in ifaces:
+            for metric_name in sys_net_metrics:
+                metric_file_location = os.path.join(sys_net_location, iface, metric_name)
+                value = self._read_int_file(metric_file_location)
+                if value is not None:
+                    self.gauge('system.net.iface.{}'.format(metric_name), value, tags=custom_tags + ["iface:" + iface])
+            iface_queues_location = os.path.join(sys_net_location, iface, 'queues')
+            self._get_num_iface_queues(iface, iface_queues_location, custom_tags)
+
+    def _get_num_iface_queues(self, iface, iface_queues_location, custom_tags):
```

Comment:
```suggestion
    def _collect_iface_queue_metrics(self, iface, iface_queues_location, custom_tags):
```
nit- can we update the name of this function to reflect that metrics are being collected?

Reply:
Updated